src/components/color-schemes-help/index.js

```diff
@@ -0,0 +1,20 @@
+import { i18n } from 'stackable'
+
+import { __ } from '@wordpress/i18n'
+import { Button } from '@wordpress/components'
+import { dispatch } from '@wordpress/data'
+
+export const ColorSchemesHelp = () => {
+	const onClick = () => {
+		// Open the global settings sidebar and the color schemes panel.
+		dispatch( 'core/edit-post' ).openGeneralSidebar( 'stackable-global-settings/sidebar' )
```

Comment:
This might error in the site editor since I think `core/edit-post` is not available there.